.travis.yml

```diff
@@ -0,0 +1,10 @@
+dist: trusty
```

Comment:
Any reason to be using such an old release of Ubuntu? Would 16.04 LTS be better? Or do we need the old one to give us old things for our old software? :)